services/web/server/src/simcore_service_webserver/garbage_collector_core.py

```diff
@@ -375,7 +376,14 @@ async def _remove_single_service_if_orphan(
                 save_state = False
             # -------------------------------------------
 
-            await director_v2_api.stop_dynamic_service(app, service_uuid, save_state)
+            try:
+                await director_v2_api.stop_dynamic_service(
```

Comment:
you can use the ```on_error``` argument here instead of that weird construction.

Reply:
Just to be sure. You used the ``on_error`` as @sanderegg suggested but I still see here the exception handled. Do you really still need that?